clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/StateRepositoryOpCodec.java

```diff
@@ -152,8 +155,26 @@ public StateRepositoryOpMessage decode(EhcacheMessageType messageType, ByteBuffe
     String mapId = decoder.string(MAP_ID_FIELD);
 
     ByteBuffer keyBuffer = decoder.byteBuffer(KEY_FIELD);
-    Object key = Util.unmarshall(keyBuffer);
+    Object key = Util.unmarshall(keyBuffer, WHITELIST_PREDICATE);
 
     return new StateRepositoryOpMessage.GetMessage(storeName, mapId, key);
   }
+
+  public static final Predicate<Class<?>> WHITELIST_PREDICATE = Collections.unmodifiableList(Arrays.asList(
+    java.lang.Integer.class,
+    java.lang.Long.class,
+    java.lang.Float.class,
+    java.lang.Double.class,
+    java.lang.Byte.class,
+    java.lang.Character.class,
+    java.lang.String.class,
+    java.lang.Boolean.class,
+    java.lang.Short.class,
+    java.lang.Void.class,
```

Comment:
`Void` isn't `Serializable` why is it here?

Reply:
removed Void from the list